### PR TITLE
JE-224: Added link to form

### PR DESCRIPTION
**Pull request by tuj**
https://jira.itkdev.dk/browse/JE-224

### CODE REVIEW

File: bundles/Expense/Resources/views/expense/new.html.twig [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 {% block content %}
-    {{ form(form) }}
+    {{ form_start(form) }}
+        {{ form_row(form.project) }}
+        {{ form_row(form.issue_key) }}
+        {{ form_row(form.category) }}
+        <div class="text-right">
+            <a href="{{ path('expense_admin_category_index') }}">{{ 'expense.category_admin'|trans }}</a>
+        </div>
+        {{ form_rest(form) }}
```
Comment (rimi-itk) [inline bundles/Expense/Resources/views/expense/new.html.twig:11]: Strictly speaking, this is not needed (`form_end` will render any not yet rendered fields), but being explicit never hurts.

**Review by rimi-itk — APPROVED** [review]
Godkendt med anmærkning.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Jira url is missing!

**Comment by tuj** [comment]
Jira link added.

File: bundles/Expense/Resources/views/expense/new.html.twig (hunk 1 above)
Comment (tuj) [reply bundles/Expense/Resources/views/expense/new.html.twig:11]: I know, but it signals that there are more pieces to the form.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**